cf-agent/package_module.c

```diff
@@ -1472,6 +1486,25 @@ PromiseResult HandlePresentPromiseAction(EvalContext *ctx,
                 return PROMISE_RESULT_FAIL;
             }
         }
+        else if (package_info->type == PACKAGE_TYPE_REPO)
+        {
+            /* We are expecting only package name to be returned by
+             * 'get-package-data' in case of repo package */
+            if (package_info->arch)
+            {
+                Log(LOG_LEVEL_VERBOSE,
+                    "Unexpected package architecture received");
+                free(package_info->arch);
+                package_info->arch = NULL;
+            }
+            if (package_info->version)
+            {
+                Log(LOG_LEVEL_VERBOSE,
+                    "Unexpected package version received");
```

Comment:
Suggest to change this to ``Unexpected package version received from package module``, just to make the context clear. Same for architecture.